.whitesource

```diff
@@ -0,0 +1,15 @@
+{
+  "scanSettings": {
+    "configMode": "LOCAL",
+    "configExternalURL": "",
+    "projectToken": "",
+    "baseBranches": ["main","stable"]
+  },
+  "checkRunSettings": {
+    "vulnerableCheckRunConclusionLevel": "failure",
+    "displayMode": "diff"
+  },
+  "issueSettings": {
+    "minSeverityLevel": "LOW"
```

Comment:
this will create a github issue for each n every issue found by this scan ? or will there be a jira tkt created

Reply:
@mukul-seagate11 please confirm.

Reply:
yes, GitHub issue will be create by default but can be integrated with Jira ticket as well